altk/pre_tool/core/types.py

```diff
@@ -119,41 +119,41 @@ class RefractionRunOutput(PreToolReflectionRunOutput):
     result: Optional[DebuggingResult] = None
 
 
-class ToolGuardBuildInputMetaData(BaseModel):
-    model_config = ConfigDict(arbitrary_types_allowed=True)
-    policy_text: str = Field(description="Text of the policy document file")
-    short1: bool = Field(default=True, description="Run build short or long version. ")
-    validating_llm_client: LLMClient = Field(
-        description="ValidatingLLMClient for build time"
-    )
+# class ToolGuardBuildInputMetaData(BaseModel):
```

Comment:
Is it important to keep these around? Otherwise just remove.

Reply:
removed